### PR TITLE
fix: aod cli subcommand

### LINKED ISSUE #60: Use "gcloud" as the subcommand to run gcloud commands
https://github.com/abcxyz/access-on-demand/blob/main/pkg/cli/root.go#L45

`cli` as the subcommand name is confusing

### PR DESCRIPTION
**Pull request by sqin2019**
fixes #60

### CODE REVIEW

**Review by mattrandallbecker — DISMISSED** [review]

**Review by yolocs — DISMISSED** [review]

File: .github/workflows/validate.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ jobs:
             echo "::notice title=IAM Request Validation::Skip because iam.yaml is not found"
           fi
           if [ -f "cli.yaml" ]; then
-            aod cli validate -path $CLI_FILE_PATH
+            aod tool validate -path $CLI_FILE_PATH
           else
             echo "::notice title=CLI Request Validation::Skip because cli.yaml is not found"
```
Comment (yolocs) [inline .github/workflows/validate.yml:68]: Should we update request file to be `tool.yaml`

File: pkg/cli/root.go
Comment (yolocs) [inline pkg/cli/root.go:1]: Can we rename the files?

File: pkg/cli/root_test.go [hunk 2]
```diff
@@ -25,8 +25,8 @@ func TestRootCommand_Help(t *testing.T) {
 	exp := `
 Usage: aod COMMAND
 
-  cli    Perform operations related to the CLI request
-  iam    Perform operations related to the IAM request
+  iam     Perform operations related to the IAM request
+  tool    Perform operations related to the CLI request
```
Comment (yolocs) [inline pkg/cli/root_test.go:29]:
```suggestion
  iam     Perform operations to modify IAM policies on demand.
  tool    Perform operations to run CLI tools on demand.
```

File: pkg/cli/root.go
Comment (sqin2019) [reply pkg/cli/root.go:1]: Renamed.

**Review by yolocs — APPROVED** [review]

**Pull request merged**